samples/excel/89-preview-apis/events-tablecollection-changed.yaml

```diff
@@ -0,0 +1,164 @@
+order: 9
+id: excel-events-tablecollection-changed
+name: Events - Table collection changed
+description: Add event handlers for table collection onDataChanged event
+host: EXCEL
+api_set:
+    ExcelApi: 1.7
+script:
+    content: |+
+        $("#setup").click(() => tryCatch(setup));
+        $("#register-on-data-changed-handler").click(() => tryCatch(registerOnDataChangedHandler));
+        $("#change-data").click(() => tryCatch(changeData));
+
+        async function registerOnDataChangedHandler() {
+            await Excel.run(async (context) => {
+                let table = context.workbook.tables;
+                table.onDataChanged.add(onDataChange);
+
+                await context.sync();
+
+                OfficeHelpers.UI.notify("A handler has been registered for the table collection onDataChanged event",
+                    "Try changing cell values in the tables, and watch the console output.");
+            });
+        }
+
+        async function changeData() {
+            await Excel.run(async (context) => {
+                let sheet = context.workbook.worksheets.getItem("Sample");
+                let range1 = sheet.getRange("B7");
+                let range2 = sheet.getRange("C15");
+                range1.values = [[900]];
+                range2.values = [[20]];
+                range1.format.autofitColumns();
+                range2.format.autofitColumns();
+
+                await context.sync();
+
+                console.log("B7 value has been changed.");
```

Comment:
Suggest that you either update this console.log message to indicate that the value of B7 *and* C15 have changed ("B7 value and C15 value have been changed.")

OR -- you could just remove this console.log altogether, since the event-handler function is already writing messages to the console to inform the user about changes to data in B7 and C15.